packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+export type Props = {
+  /** Term label describing the value displayed. */
+  label: string;
```

Comment:
Should probably allow a node, so that we can customize the label a bit (like add an icon or something).

Reply:
Or maybe add `textProps` so the label can be changed entirely? Instead of piping small/uppercased/etc.

Reply:
got it!